reflectwalk.go

```diff
@@ -4,9 +4,7 @@
 // those elements.
 package reflectwalk
 
-import (
-	"reflect"
-)
+import "reflect"
```

Comment:
ah, that's a `goimports` thing. If it's OK with you it's easier to leave it and let goimports manage that block.